src/components/ChatView.vue

```diff
@@ -19,15 +19,40 @@
   -->
 
 <template>
-	<div class="chatView">
-		<MessagesList :token="token" />
-		<NewMessageForm />
+	<div
+		class="chatView"
+		@dragover.prevent="isDraggingOver = true"
+		@dragleave.prevent="isDraggingOver = false"
+		@drop.prevent="handleDropFiles">
+		<transition name="slide" mode="out-in">
+			<div
+				v-show="isDraggingOver"
+				class="dragover">
+				<div class="drop-hint">
+					<div
+						class="drop-hint__icon"
+						:class="{'icon-upload' : !isGuest, 'icon-error' : isGuest}" />
```

Comment:
I really dislike `icon-error`, I think I would keep the `icon-user` or `icon-upload` instead.